build/windows/installer/scripts/livenessprobe.cmd

```diff
@@ -7,6 +7,13 @@ IF ERRORLEVEL 1 (
     exit /b 1
 )
 
+tasklist /fi "imagename eq telegraf.exe" /fo "table"  | findstr telegraf
+
+IF ERRORLEVEL 1 (
+    echo "Telegraf is not running"
```

Comment:
Will this also be sent to our traces telemetry, the same as for linux?

Reply:
Got rid of the windows componenet, right now only linux. Rashmi and I decided to add telegraf to the windows liveness probe later.